### DISCUSSION TITLE
Some one is trying to find my email on hit hub it’s my ex new boy friend and he’s jailbird psychopath drug addict

### DISCUSSION THREAD

**Discussion by 984mk** [community/community#53277] [Projects and Issues]
### Select Topic Area

General

### Select Feature Area

- [X] Issues
- [X] Projects

### Body

Hi my name is mike Tootoosis I have been learning and iam quit untreated in learning this for anew job in security but unfortunately I have some one who is my exs new boyfriend and he’s is on my projects trying to find my email to hurt me physically I would like some help to remove his account and acces to my builds I was trying my hand at Sherlock last night and we’ll this morning he was posing as me trying to ask questions to ppl about how to manipulate Sherlock to find me can some one help me iam genuinely worried he probly has a gun and I called the cops and they said they can’t do anything can some one help me 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @984mk 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
